src/civetweb.c

```diff
@@ -192,7 +180,25 @@ clock_gettime(int clk_id, struct timespec *t)
 	}
 	return -1; /* EINVAL - Clock ID is unknown */
 }
+
+/* if clock_gettime is declared, then __CLOCK_AVAILABILITY will be defined */
```

Comment:
This is not true. XCode's header undefs __CLOCK_AVAILABILITY after it's done with it.